### ISSUE TITLE
UI - allow interacting with post in a specific tag page

### ISSUE THREAD

**Issue by valvin1** [friendica/friendica#8225]
When I'm clicking in one of the tag I've saved I'm redirected to a search page filtered on this tag.

I can see posts but when I want to interact with it I have to click on `View in context` which requires a page load and the lost of my position in the tag's timeline.

Would it be possible to like, comment, reshare when I'm in this page without leaving it?

**Comment by MrPetovan** [comment]
Yes, this is a long-term goal to end the formal separation between public items (that you couldn’t interact with) and the user-specific items to which interactions are attached to.

**Comment by annando** [comment]
There is a conceptual problem: We do not only search in starting posts, but also in comments. Comments themselves can be out of context, so you need to see them in context.

So either we only search in starting posts, not comments or we display the whole thread because somewhere in the comments the searched tag had been used - or we stay this way.

BTW: You can simply open the context in a new browser tab to interact.
